metrics/src/vespa/metrics/metricmanager.cpp

```diff
@@ -165,9 +169,8 @@ void
 MetricManager::init(const config::ConfigUri & uri, bool startThread)
 {
     if (isInitialized()) {
-        throw vespalib::IllegalStateException(
-                "The metric manager have already been initialized. "
-                "It can only be initialized once.", VESPA_STRLOC);
+        throw IllegalStateException("The metric manager have already been initialized. "
+                                    "It can only be initialized once.", VESPA_STRLOC);
```

Comment:
Also looks like something that should be an `assert`. `IllegalStateException` always sounded like a bad idea since it implies the program execution has gone awry...

Reply:
I will do a cleanup of this afterwards.